win32ss/user/ntuser/kbdlayout.c

```diff
@@ -23,6 +23,7 @@ PKL gspklBaseLayout = NULL; /* FIXME: Please move this to pWinSta->spklList */
 PKBDFILE gpkfList = NULL;
 DWORD gSystemFS = 0;
 UINT gSystemCPCharSet = 0;
+DWORD gLCIDSentToShell = 0; /* Win: gLCIDSentToShell */
```

Comment:
If identical name, no need for the "Win" comment.

Reply:
Deleted.